frontend/vocab/constants.js

```diff
@@ -0,0 +1,6 @@
+const VOCAB_HELP_TEXT = {
```

Comment:
@shapiromatron My thought is that we can control how the namespaces are shown in different contexts with this file, similar to what we do with RoB. Let me know if you'd rather me just use `assessment.get_vocabulary_display()` and I'll remove this.

Reply:
yeah, thats what I was thinking originally too, but this is such a simple pattern and easy to track we can use it for now